lib/core/src/display/scene/camera.rs

```diff
@@ -0,0 +1,20 @@
+use super::Object;
+use nalgebra::base::Matrix4;
+use nalgebra::geometry::Perspective3;
+use std::f32::consts::PI;
+
+#[derive(Shrinkwrap)]
+#[shrinkwrap(mutable)]
+pub struct Camera {
+    #[shrinkwrap(main_field)]
+    pub object: Object,
+    pub projection: Matrix4<f32>,
+}
```

Comment:
We align `:` in struct defs. I know this is stupid to write such comments, but until we find a way to fix / modify rustfmt we need to do it by hand.

Reply:
A E S T H E T I C S is not stupid. Fixed!